packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts

```diff
@@ -39,361 +37,356 @@ import { DiscoveryOptions } from "fabric-network";
+      commit,
+      packaging,
+      queryCommitted,
+    } = lifecycle;
+
+    Checks.truthy(packageIds, `packageIds truthy OK`);
+    Checks.truthy(
+      Array.isArray(packageIds),
+      `Array.isArray(packageIds) truthy OK`,
+    );
+    Checks.truthy(approveForMyOrgList, `approveForMyOrgList truthy OK`);
+    Checks.truthy(
+      Array.isArray(approveForMyOrgList),
+      `Array.isArray(approveForMyOrgList) truthy OK`,
+    );
+    Checks.truthy(installList, `installList truthy OK`);
+    Checks.truthy(
+      Array.isArray(installList),
+      `Array.isArray(installList) truthy OK`,
+    );
+    Checks.truthy(queryInstalledList, `queryInstalledList truthy OK`);
+    Checks.truthy(
+      Array.isArray(queryInstalledList),
+      `Array.isArray(queryInstalledList) truthy OK`,
+    );
+    Checks.truthy(commit, `commit truthy OK`);
+    // Checks.truthy(init, `init truthy OK`);
+    Checks.truthy(packaging, `packaging truthy OK`);
+    Checks.truthy(queryCommitted, `queryCommitted truthy OK`);
+
+    // FIXME - without this wait it randomly fails with an error claiming that
+    // the endorsement was impossible to be obtained. The fabric-samples script
+    // does the same thing, it just waits 10 seconds for good measure so there
+    // might not be a way for us to avoid doing this, but if there is a way we
+    // absolutely should not have timeouts like this, anywhere...
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    const assetId = uuidv4();
+    const assetType = "asset";
+
+    const assetData = {
+      objectType: assetType,
+      assetID: assetId,
+      color: "gray",
+      size: 3,
+      appraisedValue: 500,
+    };
+
+    //Chaincode-specific method requires attribute asset_properties
+    const rawTmpData = {
+      asset_properties: assetData,
+    };
+
+    // CreateAsset(id string, color string, size int, owner string, appraisedValue int)
+    const createRes = await apiClient.runTransactionV1({
+      transientData: rawTmpData,
+      contractName,
+      channelName,
+      //objectType, assetID, color, size, appraisedvalue
+      params: [],
+      methodName: "CreateAsset",
+      invocationType: FabricContractInvocationType.Sendprivate,
+      signingCredential: {
+        keychainId,
+        keychainRef: keychainEntryKey,
+      },
+    });
+    expect(createRes).toBeTruthy();
+    expect(createRes.status).toBeWithin(199, 300);
+    const getRes = await apiClient.runTransactionV1({
+      contractName,
+      channelName,
+      params: [assetId],
+      methodName: "ReadAsset",
+      invocationType: FabricContractInvocationType.Send,
+      signingCredential: {
+        keychainId,
+        keychainRef: keychainEntryKey,
+      },
+    });
+
+    expect(getRes).toBeTruthy();
+    expect(getRes.data).toBeTruthy();
+    expect(getRes.data.functionOutput).toBeTruthy();
+    expect(getRes.status > 199).toBe(true);
+    expect(createRes.status < 300).toBe(true);
+    //TODO FIX:
```

Comment:
@awadhana Is thi still failing with the errors the comment is saying? If yes we need to do a deep dive.

Reply:
Yes, @petermetz still failing. I'll keep this flagged to go back to in the future.

Reply:
![image](https://user-images.githubusercontent.com/29382145/144109559-26049a96-a6d4-4237-875b-b270ef01eb2c.png)
Actually, it seems to be passing for me now. Let me know if it looks good to you too

Reply:
Also, just wanted to notify you @petermetz, everything should be solid now :)